modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -1037,15 +1041,34 @@ EFAElement3D::createChild(const std::set<EFAElement *> & CrackTipElements,
           center += cut_plane_points[i];
         center /= cut_plane_points.size();
 
+        for (unsigned int i = 0; i < cut_plane_points.size(); ++i)
```

Comment:
This whole block can be deleted if `orig_v` is not needed (as it appears is the case).